### ISSUE TITLE
Samples with multisampling at broken on devices with a devicePixelRatio != 1

### ISSUE THREAD

**Issue by toji** [austinEng/webgpu-samples#222]
This is easiest to observe on the desktop by opening up dev tools and using the Device Toolbar (Ctrl+Shift+M) to emulate a mobile-sized screen. I tested using the Pixel 5 in the drop down menu. When in this mode most examples will break with the following error:

```
The Resolve target [TextureView] size (width: 600, height: 600) does not match the color attachment [TextureView] size (width: 1650, height: 1650).
 - While validating colorAttachments[0].
 - While encoding [CommandEncoder].BeginRenderPass([RenderPassDescriptor]).
```

This appears to be a case of the canvas size using the `devicePixelRatio` (2.75 on the Pixel 5) to compute the texture size while the resolve target fails to take it into account, which means that it would be broken anywhere that the `devicePixelRatio` is not 1.

**Comment by toji** [comment]
Ah, this appears to be what #194 is trying to solve.

**Comment by toji** [comment]
Oh, and it turns out this isn't even just a dPR issue. The samples all use an outdated `context.configure()` call to set the size, which was recently removed entirely. I'm gonna fix it.

**Comment by toji** [comment]
Fixed by #223
